### ISSUE TITLE
re added center of pixel snapping 

### ISSUE THREAD

**Issue by MissLavender-LQ** [Bforartists/Bforartists#3713]
blender has an option for snapping uvs to center of pixels and it seems bforartist removes this feature or at least hides in in a different area  
bforartists still has the other snap to uv function but doesn't have the 2 check boxes for corner and center 

bforartists
![image](https://github.com/Bforartists/Bforartists/assets/51702517/f07d2066-7c90-4d32-a51c-bf2471202e34)

blender
![image](https://github.com/Bforartists/Bforartists/assets/51702517/acc512a9-42ef-4c45-a2f2-9f6a9f72502b)

tldr please bring back round to pixel tab in bforartist

**Comment by Draise14** [comment]
Oh and thanks for the report, keep em coming if you need help. Hope the answer clarified it.

**Comment by MissLavender-LQ** [comment]
oh thanks!! yeah i just need center of pixel snapping for quake 1 models :p
